src/components/JoinCommunity/index.jsx

```diff
@@ -22,6 +22,11 @@ export default function Community() {
                 <p>Join the ever-growing uptane community for updates, support, and collaboration.</p>
             </div>
             <div className={styles.community_links}>
+                <div className={styles.links}>
+                    {/* <LinkCard link="#" title="Slack" light="img/community_links/slack.svg" dark="img/community_links/slack.svg" /> */}
+                    <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="img/community_links/discord-mark-blue.svg" dark="img/community_links/discord-mark-blue.svg" />
+                    {/* <LinkCard link="#" title="Twitter" light="img/community_links/twitter.svg" dark="img/community_links/twitter.svg" /> */}
+                </div>
```

Comment:
@AmosBlack we don't need it side by side we need it upside down, 

instead of 

```
 <div className={styles.links}>
                    <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="img/community_links/discord-mark-blue.svg" dark="img/community_links/discord-mark-blue.svg" />
                    <LinkCard link="https://github.com/uptane" title="GitHub" light="img/community_links/github_light.svg" dark="img/community_links/github_dark.svg" />
                </div>
```

we only need this code, both links needs to be under ` <div className={styles.links}>` itself, we don't need a separate div